content/docs/setup/kubernetes/minimal-install/index.md

```diff
@@ -15,21 +15,17 @@ Refer to the [prerequisites](/docs/setup/kubernetes/quick-start/#prerequisites)
 
 ## Installation steps
 
-1. If using a Helm version prior to 2.10.0, install Istio's [Custom Resource Definitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions)
-via `kubectl apply`, and wait a few seconds for the CRDs to be committed in the kube-apiserver:
-
-    {{< text bash >}}
-    $ kubectl apply -f install/kubernetes/helm/istio/templates/crds.yaml
-    {{< /text >}}
-
-1. Choose one of the following two
-**mutually exclusive** options described below.
+Choose one of the following two **mutually exclusive** options described below.
 
 ### Option 1: Install with Helm via `helm template`
 
+Choose this option if your cluster doesn't have [Tiller](https://github.com/kubernetes/helm/blob/master/docs/architecture.md#components) deployed and you don't want to install it.
+
 1. Render Istio's core components to a Kubernetes manifest called `istio-minimal.yaml`:
 
     {{< text bash >}}
+    $ cat @install/kubernetes/namespace.yaml@ > $HOME/istio-minimal.yaml
+    $ cat install/kubernetes/helm/istio-init/files/crd-1* >> $HOME/istio-minimal.yaml
```

Comment:
the naming convetion should be good with just `crd-*` - no need for the extra 1.

I'd also recommend against this installation method of creating the CRDs within the manifest - as if the manifest is deleted, all custom resources will be deleted.  Ideally CRD creation should be a separate step...  This is why we have an entirely seperate chart for the work.  That said, if an operator wishes to ensure the CRDs are applied without syncronizing on the completion of isito-init, that is straightforward

```
for $I in install/kubernetes/helm/istio-init/files/crd*; do kubectl apply -f $i; done
```

Note CRDs are cluster-scoped and do not require a namespace to be created.

Reply:
1. My intention was that for minimal installation, the CRDs for `cert-manager` is not required. That's the reason why I added extra `1`.
2. Sounds good to create the CRDs in separate step, let me update that.